packages/shared/lib/services/notification/error.service.ts

```diff
@@ -0,0 +1,91 @@
+import type { Knex } from 'knex';
+import type { ActiveLog } from '@nangohq/types';
+import { Ok, Err } from '@nangohq/utils';
+import type { Result } from '@nangohq/utils';
+
+import db from '../../db/database.js';
+
+const DB_TABLE = '_nango_active_logs';
+
+type ErrorNotification = Required<Pick<ActiveLog, 'type' | 'action' | 'connection_id' | 'activity_log_id' | 'log_id' | 'active'>>;
+type SyncErrorNotification = ErrorNotification & Required<Pick<ActiveLog, 'sync_id'>>;
+
+export const errorNotificationService = {
```

Comment:
should all the variables/classes in this file be named after `activeLogs` because this is basically a model interacting with the db to save/retrieve/delete `activeLogs`

Reply:
Yeah, I had the same thought, but I concluded with is that `activeLogs` are actually pretty flexible and this service is one kind of usage of them. In theory we could have another class that uses them differently.